### PR TITLE
Define route search tab constants

### PR DESCRIPTION
**Pull request by stonevyt**

### CODE REVIEW

File: common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    val contentColorSecondaryOnDark = theme.colorPalette.grayScale.gray400
+
+    val backgroundColorDefaultOnLight = theme.colorPalette.grayScale.gray100
+    val backgroundColorDefaultOnDark = theme.colorPalette.grayScale.gray800
+
+    val backgroundColorActiveOnLight = theme.colorPalette.primary.alpha(0.1f)
+    val backgroundColorActiveOnDark = theme.colorPalette.primary.alpha(0.3f)
+
+    val titleStyle = theme.typographyScale.textM.copy(fontWeight = BasicFontWeight.Bold)
+    val subtitleStyle = theme.typographyScale.textS.copy(fontWeight = BasicFontWeight.Normal)
```
Comment (justasm) [inline common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt:18]: Can import BasicFontWeight.* to just write `Bold` and `Normal` here

**Review by justasm — APPROVED** [review]

File: common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import com.trafi.ui.theme.internal.BasicFontWeight.Normal
+import com.trafi.ui.theme.internal.dp
+
+class RouteSearchTabConstants (theme: CurrentTheme) {
+    val contentColorPrimary = theme.colorPalette.onSurface
+    val contentColorSecondaryOnLight = theme.colorPalette.grayScale.gray600
+    val contentColorSecondaryOnDark = theme.colorPalette.grayScale.gray400
+
+    val backgroundColorDefaultOnLight = theme.colorPalette.grayScale.gray100
+    val backgroundColorDefaultOnDark = theme.colorPalette.grayScale.gray800
```
Comment (mikasdo) [inline common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt:14]: gray scale is automatically inverted from theme, you only need a single constant

Comment (mikasdo) [inline common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt:14]:
```suggestion
    val defaultBackgroundColor = theme.colorPalette.grayScale.gray100
```

File: common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+import com.trafi.ui.theme.internal.*
+import com.trafi.ui.theme.internal.BasicFontWeight.Bold
+import com.trafi.ui.theme.internal.BasicFontWeight.Normal
+import com.trafi.ui.theme.internal.dp
+
+class RouteSearchTabConstants (theme: CurrentTheme) {
+    val contentColorPrimary = theme.colorPalette.onSurface
+    val contentColorSecondaryOnLight = theme.colorPalette.grayScale.gray600
+    val contentColorSecondaryOnDark = theme.colorPalette.grayScale.gray400
```
Comment (mikasdo) [inline common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt:11]: val contentColorSecondary = theme.colorPalette.grayScale.gray600

File: common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+    val backgroundColorDefaultOnLight = theme.colorPalette.grayScale.gray100
+    val backgroundColorDefaultOnDark = theme.colorPalette.grayScale.gray800
+
+    val backgroundColorActiveOnLight = theme.colorPalette.primary.alpha(0.1f)
+    val backgroundColorActiveOnDark = theme.colorPalette.primary.alpha(0.3f)
```
Comment (mikasdo) [inline common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt:17]: val activeBackgroundColor = theme.colorPalette.primary.alpha(0.1f)

**Review by mikasdo — CHANGES_REQUESTED** [review]

**Review by mikasdo — APPROVED** [review]

**Pull request merged**